products/google-nexus.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Google Nexus
+category: device
+tags: google mobile-phone tablet
+iconSlug: google
+permalink: /google-nexus
+versionCommand: "Settings -> About Phone -> Regulatory labels"
+releasePolicyLink: https://support.google.com/nexus/answer/11227897
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: true
+discontinuedColumn: true
+eolColumn: Security Updates
+customColumns:
+-   property: supportedAndroidVersions
+    position: after-release-column
+    # https://en.wikipedia.org/wiki/Google_Nexus
+    label: Supported Android Version
+    description: Supported Android versions range
+    link: https://endoflife.date/android
+
+# Android and Security are based on https://support.google.com/nexus/answer/11227897.
+releases:
+-   releaseCycle: "6p"
+    releaseLabel: "Nexus 6P"
+    releaseDate: 2015-09-29
+    support: 2017-09-01
+    eol: 2018-11-01
+    discontinued: true
+    link: https://en.wikipedia.org/wiki/Nexus_6P
+    supportedAndroidVersions: 6.0 - 8.1 # https://www.gsmarena.com/huawei_nexus_6p-7588.php
+
+-   releaseCycle: "5x"
+    releaseLabel: "Nexus 5X"
+    releaseDate: 2015-09-29
+    support: 2017-09-01
+    eol: 2018-11-01
+    discontinued: true
+    link: https://en.wikipedia.org/wiki/Nexus_5X
+    supportedAndroidVersions: 6.0 - 8.1 # https://www.gsmarena.com/lg_nexus_5x-7556.php
+
+-   releaseCycle: "player"
+    releaseLabel: "Nexus Player"
+    releaseDate: 2014-11-03
+    support: 2016-11-01
+    eol: 2016-11-01
+    discontinued: true
+    link: https://en.wikipedia.org/wiki/Nexus_Player
+    supportedAndroidVersions: 5.0 - 8.0 # https://en.wikipedia.org/wiki/Nexus_Player#
+
+-   releaseCycle: "6"
+    releaseLabel: "Nexus 6"
+    releaseDate: 2014-10-15
+    support: 2016-10-01
+    eol: 2017-10-01
+    discontinued: true
+    link: https://en.wikipedia.org/wiki/Nexus_6
+    supportedAndroidVersions: 5.0 - 7.1.1 # https://www.gsmarena.com/motorola_nexus_6-6604.php
+
+-   releaseCycle: "7"
+    releaseLabel: "Nexus 7"
+    releaseDate: 2013-07-26
+    support: 2015-07-01
+    eol: 2016-08-01
+    discontinued: true
+    link: https://en.wikipedia.org/wiki/Nexus_7_(2013)
+    supportedAndroidVersions: 4.3 - 5.1 # https://www.gsmarena.com/asus_google_nexus_7_(2013)-5600.php
```

Comment:
```suggestion
    supportedAndroidVersions: 4.3 - 6.0 # https://www.gsmarena.com/asus_google_nexus_7_(2013)-5600.php
```

According to both  https://www.gsmarena.com/asus_google_nexus_7_(2013)-5600.php and https://en.wikipedia.org/wiki/Nexus_7_(2013).